### ISSUE TITLE
Missing slug parameter in redirect when using UrlNormalizer

### ISSUE THREAD

**Issue by xr0m3oz** [codemix/yii2-localeurls#105]
The problem is that the redirect is not in the main language, it does not work correctly. If you pass a parameter, this parameter is ignored.
`site.com `
In the settings I have that it is Russian and it is by default, in the address bar the language parameter is not passed. If I go to a dynamic page for example:

`site.com/shop/name-shop`
There is a redirect to
`site.com/shop/name-shop/  `
All right. At the end, a slash is added.

If I switch the language to another not basic, then I will have an address of this kind.
`site.com/en/shop/name-shop`
There is a redirect to
`site.com/en/shop/`
And here the problem is, after the redirect, there is no parameter of the store name. name-shop

It turns out in the basic language everything works fine, not in the main language of the parmatra disappear. With a redirect without a backslash at the end.
```'urlManager' => [
			'class' => 'codemix\localeurls\UrlManager',
			'suffix' => '/',
			'normalizer' => [
				'class' => 'yii\web\UrlNormalizer',
				// use temporary redirection instead of permanent for debugging
				'action' => \yii\web\UrlNormalizer::ACTION_REDIRECT_PERMANENT,
			],
			'languages' => ['en', 'ru', 'uk'],
			'enablePrettyUrl' => true,
			'showScriptName' => false,
```

**Comment by mikehaertl** [comment]
@xr0m3oz Do you have any URL rules configured? Because I can't see how `name-shop` should be a URL parameter. Parameters by definition are things like `...?name=value`. Also show, how you created this URL.

**Comment by xr0m3oz** [comment]
Yes rules config 
```
			'rules' => [
				'/' => 'site/index',
				'shop/<slug:[\w-]+>' => 'shop/view',
			],
```
site.com/shop/name-shop   (name-shop is slug)

**Comment by mikehaertl** [comment]
@xr0m3oz I've added a fix. Can you check the latest `UrlManager.php` from the `master` branch if it fixes the problem for you?

**Comment by danaki** [comment]
We have similar issues. I've checked out dev-master with the last fix and it did help.

**Comment by mikehaertl** [comment]
Published 1.4.11 including this fix.
